src/renderer/dataframe/Dataframe.js

```diff
@@ -152,12 +152,14 @@ export default class Dataframe extends DummyDataframe {
     }
 
     getPropertyTexture (propertyName) {
-        if (this.propertyTex[propertyName]) {
-            return this.propertyTex[propertyName];
+        const encodedPropertyName = this._encodePropertyName(propertyName);
+
+        if (this.propertyTex[encodedPropertyName]) {
+            return this.propertyTex[encodedPropertyName];
         }
 
         this._loadPropertyValuesToTexture(propertyName);
```

Comment:
We can pass here `encodedPropertyName` and avoid to encode it again in the method, right? It's only called here.

Reply:
Nope, because `_loadPropertyValuesToTexture` uses `propertyName` to access the `properties` and the encodedPropertyName is only for the textures.